WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -348,6 +348,10 @@ public static boolean supportsTiledGalleryFeature(SiteModel site) {
         return site != null && site.isWPCom();
     }
 
+    public static boolean supportsVideoPressFeature(SiteModel site) {
+        return site != null && site.isWPCom();
```

Comment:
Enabled only on simple WordPress.com sites for now and added [a task to the project board](https://github.com/orgs/wordpress-mobile/projects/166/views/1?layout=board) for this to eventually be changed to match the web.